src/modules/dashboard/components/FeaturedNotificationComponent.tsx

```diff
@@ -0,0 +1,139 @@
+  const dispatch = useDispatch();
+
+  const FeaturedChannelsList = FeaturedNotificationChannelsList[appConfig.appEnv];
+
+  const { currentIndex, handleNext, handlePrevious, listRef } = useSmoothHorizontalScroll({
+    items: FeaturedChannelsList,
+    itemsPerPage,
+    itemGap: 24, // Gap provided in between the list items
+  });
+
+  const handleClick: VoidFunction = () => {
+    handleNext();
+  };
+
+  useEffect(() => {
+    if (!account || !userPushSDKInstance) return;
+    (async function () {
+      const subscriptionsArr: {
+        channel: string;
+        user_settings: string | null;
+      }[] = await userPushSDKInstance.notification.subscriptions();
+      const subscriptionsMapping: { [channel: string]: boolean } = {};
+      const userSettings: { [channel: string]: { [key: string]: any } | null } = {};
+
+      subscriptionsArr.forEach(({ channel, user_settings }) => {
+        subscriptionsMapping[channel] = true;
+        userSettings[channel] = user_settings ? JSON.parse(user_settings) : null;
+      });
+
+      dispatch(updateBulkSubscriptions(subscriptionsMapping));
+      dispatch(updateBulkUserSettings(userSettings));
+    })();
+  }, [account, userPushSDKInstance]);
+
+  return (
+    <Box
+      borderRadius="24px"
+      padding={{ ml: 's6 s4', initial: 's6' }}
+      display="flex"
+      flexDirection="column"
+      backgroundColor={{ light: 'dark-white', dark: 'gray-900' }}
+      gap={{ ml: 's4', initial: 's2' }}
+    >
+      <Box
+        display="flex"
+        justifyContent="space-between"
+        flexDirection={{ tb: 'column' }}
+        gap={{ tb: 's3' }}
+      >
+        <Text
+          variant="h4-bold"
+          color={{ light: 'black', dark: 'white' }}
+        >
+          Featured Notification Channels
+        </Text>
+
+        <Box
+          display="flex"
+          flexDirection="row"
+          alignItems="center"
+          gap="s4"
+        >
+          <Link
+            to="/channels"
+            textProps={{
+              variant: 'h5-semibold',
+              color: { light: 'black', dark: 'white' },
+            }}
+          >
+            View All
+          </Link>
+
+          <Box
+            display="flex"
+            flexDirection="row"
+          >
+            <HoverableSVG
+              onClick={handlePrevious}
+              defaultColor={mode === 'dark' ? 'gray-400' : 'gray-900'}
+              disabled={currentIndex === 0}
+              icon={<PrevIconSlider size={24} />}
+            ></HoverableSVG>
+            <HoverableSVG
+              onClick={handleClick}
+              defaultColor={mode === 'dark' ? 'gray-400' : 'gray-900'}
+              disabled={currentIndex + itemsPerPage >= FeaturedChannelsList.length}
+              icon={<NextIconSlider size={24} />}
+            ></HoverableSVG>
+          </Box>
+        </Box>
+      </Box>
+
+      <Box
+        ref={listRef}
+        display="flex"
+        flexDirection={{ initial: 'row', tb: 'column' }}
```

Comment:
@abhishek-01k this Box and the inner loop should be a part of FeaturedChannelsList which is not used anywhere.